src/main/java/com/example/loosecoupling/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package com.example.loosecoupling.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
```

Comment:
If we have an exception handler then do we need this annotation?

Reply:
no need, because I already used HttpStatus.NOT_FOUND in ResponseEntity in GlobalExceptionHandler.